internal/adapters/storage/postgres/scheduler/db_scheduler.go

```diff
@@ -113,3 +116,35 @@ func (s *Scheduler) ToScheduler() (*entities.Scheduler, error) {
 		Autoscaling:     info.Autoscaling,
 	}, nil
 }
+
+func parseFromAnnotationEntityToMap(entities []*entities.Annotations) []*map[string]string {
+	if len(entities) == 0 {
+		return nil
+	}
+	annotations := make([]*map[string]string, len(entities))
+
+	for i, annotation := range entities {
+		annotations[i] = &map[string]string{
+			annotation.Name: annotation.Opt,
+		}
+	}
+
+	return annotations
+}
+
+func parseFromMapToAnnotationEntity(annotationMap []*map[string]string) []*entities.Annotations {
+	if len(annotationMap) == 0 {
+		return nil
+	}
+
+	annotations := make([]*entities.Annotations, len(annotationMap))
+
+	for i, annotation := range annotationMap {
+		for name, opt := range *annotation {
+			annotation := entities.NewAnnotations(name, opt)
+			annotations[i] = annotation
```

Comment:
If there are many annotations, `annotations[i]` will store only the last one. Is this expected?

Reply:
Could you explain why? I don't understand, because I've tested and all annotations came as expected
![Captura de Tela 2023-05-18 às 10 25 14](https://github.com/topfreegames/maestro/assets/36175838/ed6970b6-8883-4d3a-905a-a9abec621eef)